interactions/models/internal/extension.py

```diff
@@ -145,6 +151,14 @@ def listeners(self) -> List["Listener"]:
         """Get the listeners from this Extension."""
         return self._listeners
 
+    def load(self, *args, **kwargs):
+        """Run on extension load"""
+        ...
+
+    async def a_load(self, *args, **kwargs):
```

Comment:
```suggestion
    async def async_load(self, *args, **kwargs):
```

Reply:
Extending the proposal from `load()`, choose this over and resolve if the former is applied

```suggestion
    async def async_load(self, *args, **kwargs) -> Callable[[], ...]:
```